src/style.css

```diff
@@ -1,3 +1,80 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
```suggestion
```
it is redundant to reset margins and paddings for all selectors. reset onlu for elements which need it